nervaluate/nervaluate_cli.py

```diff
@@ -0,0 +1,60 @@
+import os
+import json
+import typer
+import spacy
+
+from nervaluate import Evaluator
+
+app = typer.Typer()
+
+
+def create_prodigy_spans(doc):
+    pred = []
+    for entity in doc.ents:
+        pred.append({"label": entity.label_, "start": entity.start, "end": entity.end})
+    return pred
+
+
+def check_labels(meta):
+    labels = []
+    for item in meta:
+        if "label" in item:
+            labels.append(item["label"])
+    return labels
+
+
+@app.command()
+def nervaluate(model_path: str, data_path: str, results_path: str = "results/"):
+    spacy_model = spacy.load(model_path)
+
+    true = []
+    pred = []
+    tags = {}
+
+    with open(data_path) as f:
```

Comment:
I think we have confused @Eleni170 here. Maybe the disagreement comes from different ways we envision the tool. I suggest that nervaluate works similarly to `spacy evaluate` which means it can accept a model path and as such it should be able to accept a spacy or prodigy model to begin with.

Think about it a different way, assume we create a way to create predictions from spacy, another to convert to prodigy format, another to create predictions from prodigy. Eventually we would want to combine them into a one line command, at least for spacy / prodigy that is the same ecosystem.

If this is taking more steps than we want to do then at least let's ensure we create (some may exist) the 2 or 3 that cube or another client might need to go from a spacy model to nervaluate predictions. Something like
```
nervaluate predict spacy_model_path preditions_path(in .spacy or .json)
nervaluate convert predictions_path prodigy_pred_path
nervaluate prodigy_pred_path prodigy_true_path
```

Reply:
Yes sorry @Eleni170, I think @nsorros and I probably needed to have a discussion about our visions for the future of nervaluate here.

See #3 for where I am coming from. My vision is a tool which is can use loaders to transform arbitrary formats and calculate scores for them. And note that there are requests coming from users. One thing we may not have mentioned, is that this is a package that is on pip - you can install it with `pip install nervaluate`, so I think that our thought process needs to be wider than just creating a tool that fits our current and specific use case. This is my motivation for not forcing users to install spaCy as a dependency, if they only want to use the package to evaluate the outputs from [flair, for example](https://github.com/MantisAI/nervaluate/issues/3#issuecomment-857515149).

Reply:
```
nervaluate predict spacy_model_path preditions_path(in .spacy or .json)
```

:point_up: Perhaps there is a reason to implement this because there is not a native `predict` command in the spacy CLI, but it definitely should not be the default CLI, it should be a subcommand with spaCy set as a non-default dependency. 

````
nervaluate convert predictions_path prodigy_pred_path
```

:point_up: Why not just  use the existing API and use a custom loader? Would we also create a convert command for conll, tsv, etc? 

```
nervaluate prodigy_pred_path prodigy_true_path --loader prodigy
nervaluate conll_pred_path conll_true_path --loader conll
```

:point_up: This is how i envisaged the CLI

Reply:
perfect let's implement all of those :D

Reply:
See #49 for a longer description of i think subcommands could look